### PR TITLE
Authenticator

### PR DESCRIPTION
**Pull request by Defuera**

### CODE REVIEW

**Comment by ragnor-rs** [comment]
rationale?

File: src/main/java/io/reist/visum/view/VisumAccountAuthenticatorActivity.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+import butterknife.ButterKnife;
+import io.reist.visum.ComponentCache;
+import io.reist.visum.ComponentCacheProvider;
+import io.reist.visum.presenter.VisumPresenter;
+
+/**
+ * Created by defuera on 01/02/2016.
+ */
+public abstract class VisumAccountAuthenticatorActivity<P extends VisumPresenter> extends AccountAuthenticatorActivity implements VisumView<P> {
```
Comment (ragnor-rs) [inline src/main/java/io/reist/visum/view/VisumAccountAuthenticatorActivity.java]: JavaDocs?

**Comment by ragnor-rs** [comment]
related tasks?

**Comment by Defuera** [comment]
well, actually there were no task. I'd like to be able working saturdays without worrying about jira, pls.

**Comment by Defuera** [comment]
>rationale

**Comment by Defuera** [comment]
>rationale
yeap

**Comment by ragnor-rs** [comment]
решили, что не стоит ради 2 методов

**Pull request closed without merging**